libopenage/engine.cpp

```diff
@@ -548,4 +542,12 @@ void Engine::move_phys_camera(float x, float y, float amount) {
 	this->engine_coord_data->camgame_phys += cam_delta.to_phys3();
 }
 
+void Engine::start_game(std::unique_ptr<GameMain> game) {
+	// TODO: maybe implement a proper 1-to-1 connection
+	assert(game);
```

Comment:
Hm, if you think it's good and necessary, then feel free to add a macro that throws the `Error`. We already have very similar code for the testing subsystem here: `libopenage/testing/testing.h`. It should still be visible instantly that this isn't the same as the standard `assert`, so maybe a `ENSURE(cond, message)` is a good keyword.